### PR TITLE
add manual exactly once test

### PR DESCRIPTION
**Pull request by rmetzger**

### CODE REVIEW

**Comment by tzulitai** [comment]
Thanks a lot for this, Robert! From this code I've learnt a few tips on Flink testing too :)
I'm giving the tests a few runs myself right away before merging.

File: flink-streaming-connectors/flink-connector-kinesis/src/test/java/org/apache/flink/streaming/connectors/kinesis/exactlyonce/ExactlyOnceTest.java [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+import org.apache.flink.configuration.Configuration;
+import org.apache.flink.streaming.api.checkpoint.Checkpointed;
+import org.apache.flink.streaming.api.datastream.DataStream;
+import org.apache.flink.streaming.api.environment.StreamExecutionEnvironment;
+import org.apache.flink.streaming.api.functions.source.SourceFunction;
+import org.apache.flink.streaming.connectors.kinesis.FlinkKinesisConsumer;
+import org.apache.flink.streaming.connectors.kinesis.FlinkKinesisProducer;
+import org.apache.flink.streaming.connectors.kinesis.config.InitialPosition;
+import org.apache.flink.streaming.connectors.kinesis.config.KinesisConfigConstants;
+import org.apache.flink.streaming.connectors.kinesis.examples.ProduceIntoKinesis;
```
Comment (tzulitai) [inline flink-streaming-connectors/flink-connector-kinesis/src/test/java/org/apache/flink/streaming/connectors/kinesis/exactlyonce/ExactlyOnceTest.java:39]: This seems to be an unused import in this test.

**Comment by tzulitai** [comment]
I've run the ExactlyOnceTest a couple of times and all of them has passed successfully. ManualConsumerProducerTest has no issues either.
Works like a charm :) Merging.

File: flink-streaming-connectors/flink-connector-kinesis/src/test/java/org/apache/flink/streaming/connectors/kinesis/exactlyonce/ExactlyOnceTest.java (hunk 1 above)
Comment (tzulitai) [inline flink-streaming-connectors/flink-connector-kinesis/src/test/java/org/apache/flink/streaming/connectors/kinesis/exactlyonce/ExactlyOnceTest.java:39]: This is quite trivial, so I'll just simply fix this import after merge.

Comment (rmetzger) [inline flink-streaming-connectors/flink-connector-kinesis/src/test/java/org/apache/flink/streaming/connectors/kinesis/exactlyonce/ExactlyOnceTest.java:39]: Thank you!

**Pull request merged**